components/compliance-service/reporting/relaxting/migrations.go

```diff
@@ -554,7 +554,7 @@ func migrateTimeSeriesDate(ctx context.Context, esClient *elastic.Client, dateTo
 				reportIds[i] = hit.Id
 			}
 
-			esInSpecReports, _ := getReportsA2v2(esClient, ctx, srcRepIndex, srcRepType, reportIds)
+			esInSpecReports, err := getReportsA2v2(esClient, ctx, srcRepIndex, srcRepType, reportIds)
```

Comment:
This one makes me nervous, if this function returns an error suddenly this entire function will error out when it wasn't before. Thoughts?

Reply:
The function returns `nil` for esInSpecReports for any non-nil error as far as I can tell.  So, provided we typically do have hits, then an error from here would likely already error out the function at line 595 if I'm reading this correctly, so I think we are probably OK here.